<!DOCTYPE html>
<html>

  <head>
    <meta charset="utf-8">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta http-equiv="X-UA-Compatible" content="IE=edge">

<title>Amin Saberi</title>
<meta name="description" content="A simple, whitespace theme for academics. Based on [*folio](https://github.com/bogoli/-folio) design.
">

<!-- Bootstrap & MDB -->
<link href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/mdbootstrap/4.17.0/css/mdb.min.css" integrity="sha256-/SwJ2GDcEt5382i8zqDwl36VJGECxEoIcBIuoLmLR4g=" crossorigin="anonymous" />

<!-- Fonts & Icons -->
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.13.0/css/all.min.css"  integrity="sha256-h20CPZ0QyXlBuAw7A+KluUYx/3pK+c7lYEpqLTlxjYQ=" crossorigin="anonymous">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/academicons/1.8.6/css/academicons.min.css" integrity="sha256-uFVgMKfistnJAfoCUQigIl+JfUaP47GrRKjf6CTPVmw=" crossorigin="anonymous">
<link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons">

<!-- Styles -->
<link rel="shortcut icon" href="/assets/img/favicon.ico">
<link rel="stylesheet" href="/assets/css/main.css">

<link rel="canonical" href="/">

<!-- Open Graph -->



  </head>

  <body class="fixed-top-nav ">

    <!-- Header -->

    <header>

    <!-- Nav Bar -->
    <nav id="navbar" class="navbar navbar-light bg-white navbar-expand-sm fixed-top">
    <div class="container">
      
        <!-- Social Icons -->
        <div class="row ml-1 ml-sm-0">
          <span class="contact-icon text-center">
  <a href="mailto:%61%6D%6E%73%62%72@%67%6D%61%69%6C.%63%6F%6D"><i class="fas fa-envelope"></i></a>
  
  <a href="https://scholar.google.com/citations?user=kv7xdfYAAAAJ&hl=en" target="_blank" title="Google Scholar"><i class="ai ai-google-scholar"></i></a>
  
  <a href="https://github.com/amnsbr" target="_blank" title="GitHub"><i class="fab fa-github"></i></a>
  <a href="https://www.linkedin.com/in/amnsbr" target="_blank" title="LinkedIn"><i class="fab fa-linkedin"></i></a>
  <a href="https://twitter.com/amnsbr" target="_blank" title="Twitter"><i class="fab fa-twitter"></i></a>
  <a href="https://www.researchgate.net/profile/Amin_Saberi2/" target="_blank" title="ResearchGate"><i class="ai ai-researchgate"></i></a>
  
  
  
  
</span>

        </div>
      
      <!-- Navbar Toogle -->
      <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar top-bar"></span>
        <span class="icon-bar middle-bar"></span>
        <span class="icon-bar bottom-bar"></span>
      </button>
      <div class="collapse navbar-collapse text-right" id="navbarNav">
        <ul class="navbar-nav ml-auto flex-nowrap">
          <!-- About -->
          <li class="nav-item active">
            <a class="nav-link" href="/">
              about
              
                <span class="sr-only">(current)</span>
              
            </a>
          </li>
          
          <!-- Other pages -->
          
          
          
          
          
          
          
          
          
          <li class="nav-item ">
              <a class="nav-link" href="/experience/">
                experience
                
              </a>
          </li>
          
          
          
          
          
          <li class="nav-item ">
              <a class="nav-link" href="/publications/">
                publications
                
              </a>
          </li>
          
          
          
          <li class="nav-item ">
              <a class="nav-link" href="/code/">
                code
                
              </a>
          </li>
          
          
          <!-- PDF CV -->
          <li class="nav-item">
              <a class="nav-link" href="/assets/pdf/Amin_Saberi_CV.pdf">
                <i class="far fa-file-pdf"></i>cv
              </a>
          </li>
        </ul>
      </div>
    </div>
  </nav>

</header>


    <!-- Content -->

    <div class="container mt-5">
      <div class="post">

  <header class="post-header">
    
    
    
    
    <a class="navbar-brand title font-weight-lighter" href="/">
     <span class="font-weight-bold">Amin</span>   Saberi
    </a>
  </header>

  <article>
    <div class="row">
      <div class="col">
        <p>I am a postdoctoral researcher at the <a href="https://cng-lab.github.io">Cognitive Neurogenetics Group</a> led by <a href="https://www.cbs.mpg.de/person/valk/1433352">Dr. Sofie Valk</a> at the <a href="http://www.cbs.mpg.de/">Max Planck Institute for Human Cognitive and Brain Sciences</a> and <a href="https://www.fz-juelich.de/inm/inm-7/EN/Home/home_node.html">Research Center Jülich (INM-7)</a>.</p>

<p>My research is focused on using computational neuroscience techniques on neuroimaging data to study the low-dimensional functional and structural properties of the brain and their links to behavior and psychiatric disorders. I am also interested in open and reproducible science, as well as developing and improving tools for analyzing brain data, with a focus on computational modeling and meta-analyses.</p>

<p>Previously, I have studied medicine at the <a href="https://www.mums.ac.ir/index.php/en/">Mashhad University of Medical Sciences</a> (2011-2019), had a research stay at the <a href="http://en.sbu.ac.ir/Research_Institutes/MedicalSci/Pages/default.aspx">Institute of Medical Science and Technology at Shahid Beheshti University</a> (2019-2021) under the supervision of <a href="https://www.fz-juelich.de/SharedDocs/Personen/INM/INM-7/EN/Tahmasian_m.html">Dr. Masoud Tahmasian</a>, and completed my PhD at the <a href="https://www.cbs.mpg.de/independent-research-groups/neurobiosocial">Cognitive Neurogenetics Group</a> (2021-2025) under the supervision of <a href="https://www.cbs.mpg.de/person/valk/1433352">Dr. Sofie Valk</a> and Prof. Simon Eickhoff.</p>

      </div>
        
        <div class="profile col-4 ">
          
            <img class="img-fluid z-depth-1 rounded" src="/assets/img/prof_pic.jpg">
          
          
        </div>
        
    </div>

    

    <div class="publications-auto" markdown="1">
    <h2>selected publications</h2>
    <ol class="bibliography"><li><div class="row">
  <!--
  <div class="col-sm-2 abbr">
  
  </div>
-->

  <div id="saberiCuBNMGPUAcceleratedBrain2025" class="col-sm-12">
    
      <span class="title">cuBNM: GPU-Accelerated Brain Network Modeling</span>
      
      <span class="author">
        
          
          
          
            
              
                <em>Saberi A</em><sup>*</sup>,
              
            
          

        
          
          
          
            
              
                
                  Wan B,
                
              
            
          

        
          
          
          
            
              
                
                  Wischnewski K,
                
              
            
          

        
          
          
          
            
              
                
                  Jung K,
                
              
            
          

        
          
          
          
            
              
                
                  Sasse L,
                
              
            
          

        
          
          
          
            
              
                
                  Hoffstaedter F,
                
              
            
          

        
          
          
          
            
              
                
                  Bernhardt B,
                
              
            
          

        
          
          
          
            
              
                
                  Eickhoff S,
                
              
            
          

        
          
          
          
            
              
                
                  Popovych O,
                
              
            
          

        
          
          
          
            
              
                
                  and Valk S<sup>*</sup>
                
              
            
          

        
      </span>
      
      <span class="periodical">
      
        <span class="journal-title">bioRxiv</span>
      
      
      <strong class="date">2025</strong>
      
      </span>
    

    

    <span class="links">
    
      [<a class="abstract">abstract</a>]
    
    <!--  -->
    
      [<a href="https://doi.org/10.1101/2025.11.13.688224" target="_blank">url</a>]
    

    
<!--     
 -->    
    
    
      [<a href="https://www.biorxiv.org/content/10.1101/2025.11.13.688224v1.full.pdf" target="_blank">pdf</a>]
    
    
    
    
    
    
    
    </span>

    <!-- Hidden abstract block -->
    
    <span class="abstract hidden">
      <p>Brain network modeling uses computer simulations to infer about latent neural properties at micro- and mesoscales by fitting brain dynamic models to empirical data of individual subjects or groups. However, computational costs of (individualized) model fitting is a major bottleneck, limiting the practical feasibility of this approach to larger cohorts and more complex models, and highlighting the need for scalable simulation implementations. Here, we introduce cuBNM, a Python package which leverages parallel processing of graphics processing units to massively accelerate simulations of brain network models. We show running simulations on graphics processing units is several hundred times faster compared to central processing units. We demonstrate the usage of cuBNM by running optimization of group-level and individualized low- and high-dimensional models. As examples of the utility of individualized models, we investigated test-retest reliability and heritability of simulated and empirical measures in the Human Connectome Project dataset, and found simulated features to be fairly reliable and heritable. Overall, cuBNM provides an efficient framework for large-scale simulations of brain network models, facilitating investigations of latent neural processes across larger cohorts, denser networks, and higher-dimensional models, which were previously less feasible in practice.</p>
    </span>
    

    <!--  -->


  </div>
</div>
</li>
<li><div class="row">
  <!--
  <div class="col-sm-2 abbr">
  
  </div>
-->

  <div id="magielseBiasaccountingMetaanalyticApproach2025" class="col-sm-12">
    
      <span class="title">A bias-accounting meta-analytic approach refines and expands the cerebellar behavioral topography</span>
      
      <span class="author">
        
          
          
          
            
              
                
                  Magielse N<sup>*</sup>,
                
              
            
          

        
          
          
          
            
              
                
                  Manoli A,
                
              
            
          

        
          
          
          
            
              
                
                  Eickhoff S,
                
              
            
          

        
          
          
          
            
              
                
                  Fox P,
                
              
            
          

        
          
          
          
            
              
                <em>Saberi A</em><sup>†</sup>,
              
            
          

        
          
          
          
            
              
                
                  and Valk S<sup>*</sup><sup>†</sup>
                
              
            
          

        
      </span>
      
      <span class="periodical">
      
        <span class="journal-title">Neuroscience & Biobehavioral Reviews</span>
        

      
      
      <strong class="date">2025</strong>
      
      </span>
    

    

    <span class="links">
    
      [<a class="abstract">abstract</a>]
    
    <!--  -->
    
      [<a href="https://doi.org/10.1016/j.neubiorev.2025.106418" target="_blank">url</a>]
    

    
<!--     
 -->    
    
    
      [<a href="https://www.sciencedirect.com/science/article/pii/S0149763425004191/pdfft?md5=bce486904a592c8c19cf45f21f6f622a&pid=1-s2.0-S0149763425004191-main.pdf" target="_blank">pdf</a>]
    
    
    
    
    
    
    
    </span>

    <!-- Hidden abstract block -->
    
    <span class="abstract hidden">
      <p>The cerebellum plays important roles in motor, cognitive, and emotional behaviors. Previous cerebellar coordinate-based meta-analyses (CBMAs) have complemented precision-mapping and parcellation approaches by finding generalizable cerebellar activations across the largest possible set of behaviors. However, cerebellar CBMAs face challenges due to inherent methodological limitations, exacerbated by historical cerebellar neglect in neuroimaging studies. Here, we show overrepresentation of superior activations, rendering the null hypothesis of standard activation likelihood estimation (ALE) unsuitable. Our new method, cerebellum-specific ALE (C-SALE), finds behavioral convergence beyond baseline activation rates. It does this by testing experimental activations versus null models sampled from a data-driven probability distribution of finding activations at any cerebellar location. Task-specific mappings in the BrainMap meta-analytic database illustrated improved specificity of the new method. Multiple (sub)domains reached convergence in specific cerebellar subregions, supporting dual motor representations and placing cognition in posterior-lateral regions. We show our method and findings are replicable using the NeuroSynth database. Across both databases, 54/138 task domains or behavioral terms, including sustained attention, somesthesis, inference, anticipation and rhythm, reached convergence in specific cerebellar subgregions. Our meta-analyic maps largely corresponded with cerebellar atlases but also showed many complementary mappings. Repeated subsampling analysis showed that motor behaviors, and to a lesser extent language and working memory, mapped to especially consistent cerebellar subregions. Lastly, we found that cerebellar clusters were parts of brain-wide coactivation networks with cortical and subcortical regions implied in these behaviors. Together, our method further complements and expands understanding of cerebellar involvement in human behavior, highlighting regions for future investigation in both basic and clinical applications.</p>
    </span>
    

    <!--  -->


  </div>
</div>
</li>
<li><div class="row">
  <!--
  <div class="col-sm-2 abbr">
  
  </div>
-->

  <div id="saberiAdolescentMaturationCortical2025" class="col-sm-12">
    
      <span class="title">Adolescent maturation of cortical excitation-inhibition ratio based on individualized biophysical network modeling</span>
      
      <span class="author">
        
          
          
          
            
              
                <em>Saberi A</em>,
              
            
          

        
          
          
          
            
              
                
                  Wischnewski K,
                
              
            
          

        
          
          
          
            
              
                
                  Jung K,
                
              
            
          

        
          
          
          
            
              
                
                  Lotter L,
                
              
            
          

        
          
          
          
            
              
                
                  Schaare H,
                
              
            
          

        
          
          
          
            
              
                
                  Banaschewski T,
                
              
            
          

        
          
          
          
            
              
                
                  Barker G,
                
              
            
          

        
          
          
          
            
              
                
                  Bokde A,
                
              
            
          

        
          
          
          
            
              
                
                  Desrivières S,
                
              
            
          

        
          
          
          
            
              
                
                  Flor H,
                
              
            
          

        
          
          
          
            
              
                
                  Grigis A,
                
              
            
          

        
          
          
          
            
              
                
                  Garavan H,
                
              
            
          

        
          
          
          
            
              
                
                  Gowland P,
                
              
            
          

        
          
          
          
            
              
                
                  Heinz A,
                
              
            
          

        
          
          
          
            
              
                
                  Brühl R,
                
              
            
          

        
          
          
          
            
              
                
                  Martinot J,
                
              
            
          

        
          
          
          
            
              
                
                  Martinot M,
                
              
            
          

        
          
          
          
            
              
                
                  Artiges E,
                
              
            
          

        
          
          
          
            
              
                
                  Nees F,
                
              
            
          

        
          
          
          
            
              
                
                  Papadopoulos Orfanos D,
                
              
            
          

        
          
          
          
            
              
                
                  Lemaitre H,
                
              
            
          

        
          
          
          
            
              
                
                  Poustka L,
                
              
            
          

        
          
          
          
            
              
                
                  Hohmann S,
                
              
            
          

        
          
          
          
            
              
                
                  Holz N,
                
              
            
          

        
          
          
          
            
              
                
                  Baeuchl C,
                
              
            
          

        
          
          
          
            
              
                
                  Smolka M,
                
              
            
          

        
          
          
          
            
              
                
                  Vaidya N,
                
              
            
          

        
          
          
          
            
              
                
                  Walter H,
                
              
            
          

        
          
          
          
            
              
                
                  Whelan R,
                
              
            
          

        
          
          
          
            
              
                
                  Schumann G,
                
              
            
          

        
          
          
          
            
              
                
                  IMAGEN CONSORTIUM ,
                
              
            
          

        
          
          
          
            
              
                
                  Paus T,
                
              
            
          

        
          
          
          
            
              
                
                  Dukart J,
                
              
            
          

        
          
          
          
            
              
                
                  Bernhardt B,
                
              
            
          

        
          
          
          
            
              
                
                  Popovych O,
                
              
            
          

        
          
          
          
            
              
                
                  Eickhoff S,
                
              
            
          

        
          
          
          
            
              
                
                  and Valk S<sup>*</sup>
                
              
            
          

        
      </span>
      
      <span class="periodical">
      
        <span class="journal-title">Science Advances</span>
        

      
      
      <strong class="date">2025</strong>
      
      </span>
    

    

    <span class="links">
    
      [<a class="abstract">abstract</a>]
    
    <!--  -->
    
      [<a href="https://doi.org/10.1126/sciadv.adr8164" target="_blank">url</a>]
    

    
<!--     
 -->    
    
    
      [<a href="https://www.science.org/doi/epdf/10.1126/sciadv.adr8164" target="_blank">pdf</a>]
    
    
    
    
    
    
    
    </span>

    <!-- Hidden abstract block -->
    
    <span class="abstract hidden">
      <p>The excitation-inhibition ratio is a key functional property of cortical microcircuits which changes throughout an individual’s lifespan. Adolescence is considered a critical period for maturation of excitation-inhibition ratio. This has primarily been observed in animal studies. However, there is limited human in vivo evidence for maturation of excitation-inhibition ratio at the individual level. Here, we developed an individualized in vivo marker of regional excitation-inhibition ratio in human adolescents, estimated using large-scale simulations of biophysical network models fitted to resting-state functional imaging data from both cross-sectional (n = 752) and longitudinal (n = 149) cohorts. In both datasets, we found a widespread decrease in excitation-inhibition ratio in association areas, paralleled by an increase or lack of change in sensorimotor areas. This developmental pattern was aligned with multiscale markers of sensorimotor-association differentiation. Although our main findings were robust across alternative modeling configurations, we observed local variations, highlighting the importance of methodological choices for future studies.</p>
    </span>
    

    <!--  -->


  </div>
</div>
</li>
<li><div class="row">
  <!--
  <div class="col-sm-2 abbr">
  
  </div>
-->

  <div id="saberiConvergentFunctionalEffects2024" class="col-sm-12">
    
      <span class="title">Convergent functional effects of antidepressants in major depressive disorder: a neuroimaging meta-analysis</span>
      
      <span class="author">
        
          
          
          
            
              
                <em>Saberi A</em>,
              
            
          

        
          
          
          
            
              
                
                  Ebneabbasi A,
                
              
            
          

        
          
          
          
            
              
                
                  Rahimi S,
                
              
            
          

        
          
          
          
            
              
                
                  Sarebannejad S,
                
              
            
          

        
          
          
          
            
              
                
                  Sen Z,
                
              
            
          

        
          
          
          
            
              
                
                  Graf H,
                
              
            
          

        
          
          
          
            
              
                
                  Walter M,
                
              
            
          

        
          
          
          
            
              
                
                  Sorg C,
                
              
            
          

        
          
          
          
            
              
                
                  Camilleri J,
                
              
            
          

        
          
          
          
            
              
                
                  Laird A,
                
              
            
          

        
          
          
          
            
              
                
                  Fox P,
                
              
            
          

        
          
          
          
            
              
                
                  Valk S,
                
              
            
          

        
          
          
          
            
              
                
                  Eickhoff S,
                
              
            
          

        
          
          
          
            
              
                
                  and Tahmasian M<sup>*</sup>
                
              
            
          

        
      </span>
      
      <span class="periodical">
      
        <span class="journal-title">Molecular Psychiatry</span>
        

      
      
      <strong class="date">2024</strong>
      
      </span>
    

    

    <span class="links">
    
      [<a class="abstract">abstract</a>]
    
    <!--  -->
    
      [<a href="https://doi.org/10.1038/s41380-024-02780-6" target="_blank">url</a>]
    

    
<!--     
 -->    
    
    
      [<a href="https://www.nature.com/articles/s41380-024-02780-6.pdf" target="_blank">pdf</a>]
    
    
    
    
    
    
    
    </span>

    <!-- Hidden abstract block -->
    
    <span class="abstract hidden">
      <p>Neuroimaging studies have provided valuable insights into the macroscale impacts of antidepressants on brain functions in patients with major depressive disorder. However, the findings of individual studies are inconsistent. Here, we aimed to provide a quantitative synthesis of the literature to identify convergence of the reported findings at both regional and network levels and to examine their associations with neurotransmitter systems.</p>
    </span>
    

    <!--  -->


  </div>
</div>
</li>
<li><div class="row">
  <!--
  <div class="col-sm-2 abbr">
  
  </div>
-->

  <div id="lotterRegionalPatternsHuman2024" class="col-sm-12">
    
      <span class="title">Regional patterns of human cortex development correlate with underlying neurobiology</span>
      
      <span class="author">
        
          
          
          
            
              
                
                  Lotter L,
                
              
            
          

        
          
          
          
            
              
                <em>Saberi A</em>,
              
            
          

        
          
          
          
            
              
                
                  Hansen J,
                
              
            
          

        
          
          
          
            
              
                
                  Misic B,
                
              
            
          

        
          
          
          
            
              
                
                  Paquola C,
                
              
            
          

        
          
          
          
            
              
                
                  Barker G,
                
              
            
          

        
          
          
          
            
              
                
                  Bokde A,
                
              
            
          

        
          
          
          
            
              
                
                  Desrivières S,
                
              
            
          

        
          
          
          
            
              
                
                  Flor H,
                
              
            
          

        
          
          
          
            
              
                
                  Grigis A,
                
              
            
          

        
          
          
          
            
              
                
                  Garavan H,
                
              
            
          

        
          
          
          
            
              
                
                  Gowland P,
                
              
            
          

        
          
          
          
            
              
                
                  Heinz A,
                
              
            
          

        
          
          
          
            
              
                
                  Brühl R,
                
              
            
          

        
          
          
          
            
              
                
                  Martinot J,
                
              
            
          

        
          
          
          
            
              
                
                  Paillère M,
                
              
            
          

        
          
          
          
            
              
                
                  Artiges E,
                
              
            
          

        
          
          
          
            
              
                
                  Papadopoulos Orfanos D,
                
              
            
          

        
          
          
          
            
              
                
                  Paus T,
                
              
            
          

        
          
          
          
            
              
                
                  Poustka L,
                
              
            
          

        
          
          
          
            
              
                
                  Hohmann S,
                
              
            
          

        
          
          
          
            
              
                
                  Fröhner J,
                
              
            
          

        
          
          
          
            
              
                
                  Smolka M,
                
              
            
          

        
          
          
          
            
              
                
                  Vaidya N,
                
              
            
          

        
          
          
          
            
              
                
                  Walter H,
                
              
            
          

        
          
          
          
            
              
                
                  Whelan R,
                
              
            
          

        
          
          
          
            
              
                
                  Schumann G,
                
              
            
          

        
          
          
          
            
              
                
                  Nees F,
                
              
            
          

        
          
          
          
            
              
                
                  Banaschewski T,
                
              
            
          

        
          
          
          
            
              
                
                  Eickhoff S,
                
              
            
          

        
          
          
          
            
              
                
                  and Dukart J<sup>*</sup>
                
              
            
          

        
      </span>
      
      <span class="periodical">
      
        <span class="journal-title">Nature Communications</span>
        

      
      
      <strong class="date">2024</strong>
      
      </span>
    

    

    <span class="links">
    
      [<a class="abstract">abstract</a>]
    
    <!--  -->
    
      [<a href="https://doi.org/10.1038/s41467-024-52366-7" target="_blank">url</a>]
    

    
<!--     
 -->    
    
    
      [<a href="https://www.nature.com/articles/s41467-024-52366-7.pdf" target="_blank">pdf</a>]
    
    
    
    
    
    
    
    </span>

    <!-- Hidden abstract block -->
    
    <span class="abstract hidden">
      <p>Human brain morphology undergoes complex changes over the lifespan. Despite recent progress in tracking brain development via normative models, current knowledge of underlying biological mechanisms is highly limited. We demonstrate that human cortical thickness development and aging trajectories unfold along patterns of molecular and cellular brain organization, traceable from population-level to individual developmental trajectories. During childhood and adolescence, cortex-wide spatial distributions of dopaminergic receptors, inhibitory neurons, glial cell populations, and brain-metabolic features explain up to 50% of the variance associated with a lifespan model of regional cortical thickness trajectories. In contrast, modeled cortical thickness change patterns during adulthood are best explained by cholinergic and glutamatergic neurotransmitter receptor and transporter distributions. These relationships are supported by developmental gene expression trajectories and translate to individual longitudinal data from over 8000 adolescents, explaining up to 59% of developmental change at cohort- and 18% at single-subject level. Integrating neurobiological brain atlases with normative modeling and population neuroimaging provides a biologically meaningful path to understand brain development and aging in living humans.</p>
    </span>
    

    <!--  -->


  </div>
</div>
</li>
<li><div class="row">
  <!--
  <div class="col-sm-2 abbr">
  
  </div>
-->

  <div id="saberi_regional_2023" class="col-sm-12">
    
      <span class="title">The regional variation of laminar thickness in the human isocortex is related to cortical hierarchy and interregional connectivity</span>
      
      <span class="author">
        
          
          
          
            
              
                <em>Saberi A</em>,
              
            
          

        
          
          
          
            
              
                
                  Paquola C,
                
              
            
          

        
          
          
          
            
              
                
                  Wagstyl K,
                
              
            
          

        
          
          
          
            
              
                
                  Hettwer M,
                
              
            
          

        
          
          
          
            
              
                
                  Bernhardt B,
                
              
            
          

        
          
          
          
            
              
                
                  Eickhoff S,
                
              
            
          

        
          
          
          
            
              
                
                  and Valk S<sup>*</sup>
                
              
            
          

        
      </span>
      
      <span class="periodical">
      
        <span class="journal-title">PLOS Biology</span>
        

      
      
      <strong class="date">2023</strong>
      
      </span>
    

    

    <span class="links">
    
      [<a class="abstract">abstract</a>]
    
    <!--  -->
    
      [<a href="https://doi.org/10.1371/journal.pbio.3002365" target="_blank">url</a>]
    

    
<!--     
 -->    
    
    
      [<a href="https://journals.plos.org/plosbiology/article/file?id=10.1371/journal.pbio.3002365&type=printable" target="_blank">pdf</a>]
    
    
    
    
    
    
    
    </span>

    <!-- Hidden abstract block -->
    
    <span class="abstract hidden">
      <p>The human isocortex consists of tangentially organized layers with unique cytoarchitectural properties. These layers show spatial variations in thickness and cytoarchitecture across the neocortex, which is thought to support function through enabling targeted corticocortical connections. Here, leveraging maps of the 6 cortical layers based on 3D human brain histology, we aimed to quantitatively characterize the systematic covariation of laminar structure in the cortex and its functional consequences. After correcting for the effect of cortical curvature, we identified a spatial pattern of changes in laminar thickness covariance from lateral frontal to posterior occipital regions, which differentiated the dominance of infra- versus supragranular layer thickness. Corresponding to the laminar regularities of cortical connections along cortical hierarchy, the infragranular-dominant pattern of laminar thickness was associated with higher hierarchical positions of regions, mapped based on resting-state effective connectivity in humans and tract-tracing of structural connections in macaques. Moreover, we show that regions with similar laminar thickness patterns have a higher likelihood of structural connections and strength of functional connections. In sum, here, we characterize the organization of laminar thickness in the human isocortex and its association with cortico-cortical connectivity, illustrating how laminar organization may provide a foundational principle of cortical function.</p>
    </span>
    

    <!--  -->


  </div>
</div>
</li></ol>
</div>

    
    <div class="social">
      <span class="contact-icon text-center">
  <a href="mailto:%61%6D%6E%73%62%72@%67%6D%61%69%6C.%63%6F%6D"><i class="fas fa-envelope"></i></a>
  
  <a href="https://scholar.google.com/citations?user=kv7xdfYAAAAJ&hl=en" target="_blank" title="Google Scholar"><i class="ai ai-google-scholar"></i></a>
  
  <a href="https://github.com/amnsbr" target="_blank" title="GitHub"><i class="fab fa-github"></i></a>
  <a href="https://www.linkedin.com/in/amnsbr" target="_blank" title="LinkedIn"><i class="fab fa-linkedin"></i></a>
  <a href="https://twitter.com/amnsbr" target="_blank" title="Twitter"><i class="fab fa-twitter"></i></a>
  <a href="https://www.researchgate.net/profile/Amin_Saberi2/" target="_blank" title="ResearchGate"><i class="ai ai-researchgate"></i></a>
  
  
  
  
</span>

      <div class="contact-note"></div>
    </div>
    
  </article>

</div>

    </div>

    <!-- Footer -->

    
<footer class="fixed-bottom">
  <div class="container mt-0 text-center">
    Powered by <a href="http://jekyllrb.com/" target="_blank">Jekyll</a> with <a href="https://github.com/alshedivat/al-folio">al-folio</a> theme. Hosted by <a href="https://pages.github.com/" target="_blank">GitHub Pages</a>.

    
    Last updated: November 2025.
    
  </div>
</footer>



  </body>

  <!-- Load Core and Bootstrap JS -->
<script src="https://code.jquery.com/jquery-3.4.1.slim.min.js" integrity="sha384-J6qa4849blE2+poT4WnyKhv5vZF5SrPo0iEjwBvKU7imGFAV0wwj1yYfoRSJoZ+n" crossorigin="anonymous"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/2.4.0/umd/popper.min.js" integrity="sha256-OH05DFHUWzr725HmuHo3pnuvUUn+TJuj8/Qz9xytFEw=" crossorigin="anonymous"></script>
<script src="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.min.js" integrity="sha384-wfSDF2E50Y2D1uUdj0O3uMBJnjuUD4Ih7YwaYd1iqfktj0Uod8GCExl3Og8ifwB6" crossorigin="anonymous"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mdbootstrap/4.17.0/js/mdb.min.js"  integrity="sha384-wfSDF2E50Y2D1uUdj0O3uMBJnjuUD4Ih7YwaYd1iqfktj0Uod8GCExl3Og8ifwB6" crossorigin="anonymous"></script>

<!-- Load Common JS -->
<script src="/assets/js/common.js"></script>

<!-- Code Syntax Highlighting -->
<link rel="stylesheet" href="https://gitcdn.link/repo/jwarby/jekyll-pygments-themes/master/github.css" />


<!-- Load KaTeX -->
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/KaTeX/0.11.1/katex.min.css" integrity="sha256-V8SV2MO1FUb63Bwht5Wx9x6PVHNa02gv8BgH/uH3ung=" crossorigin="anonymous" />
<script src="https://cdnjs.cloudflare.com/ajax/libs/KaTeX/0.11.1/katex.min.js" integrity="sha256-F/Xda58SPdcUCr+xhSGz9MA2zQBPb0ASEYKohl8UCHc=" crossorigin="anonymous"></script>
<script src="/assets/js/katex.js"></script>



<!-- Load Mansory & imagesLoaded -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/masonry/4.2.2/masonry.pkgd.min.js" integrity="" crossorigin="anonymous"></script>
<script src="https://unpkg.com/imagesloaded@4/imagesloaded.pkgd.min.js"></script>

<!-- Project Cards Layout -->
<script type="text/javascript">
  // Init Masonry
  var $grid = $('.grid').masonry({
    gutter: 10,
    horizontalOrder: true,
    itemSelector: '.grid-item',
  });
  // layout Masonry after each image loads
  $grid.imagesLoaded().progress( function() {
    $grid.masonry('layout');
  });
</script>



<!-- Enable Tooltips -->
<script type="text/javascript">
$(function () {
  $('[data-toggle="tooltip"]').tooltip()
})
</script>



<!-- Google Analytics -->
<script>
(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
})(window,document,'script','//www.google-analytics.com/analytics.js','ga');

ga('create', 'UA-174280442-1', 'auto');
ga('send', 'pageview');
</script>



</html>
